package.json

```diff
@@ -3,27 +3,30 @@
   "description": "Front end application for the Digital Marketplace",
   "version": "0.0.1",
   "private": true,
-  "engine": "node >= 4.2",
+  "engine": "node >= 6.10.3",
```

Comment:
Should this be `node == 6.10.3`?

Reply:
Yes, well spotted! I think we might want to start using version ranges such as ```6.1.x``` but we should discuss the pros and cons.